File: Backend/src/main/java/com/example/backend/HesObject.java
package com.example.backend;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class HesObject {

    // Properites
    private String hesCode;
    private boolean hesStatus;
    private String lastHesCodeChangeDate;

    public HesObject(){

    }

    public HesObject(String hesCode, boolean hesStatus, String lastHesCodeChangeDate){
        this.hesCode = hesCode;
        this.hesStatus = hesStatus;
        this.lastHesCodeChangeDate = lastHesCodeChangeDate;
    }

}

File: Frontend/src/classess/HesObject.ts
class HesObject {
  private _hesCode: string;
  private _hesStatus: boolean;
  private _lastHesCodeChangeDate: string;

  public constructor(hesCode: string, hesStatus: boolean, lastHesCodeChangeDate: string) {
    this._hesCode = hesCode;
    this._hesStatus = hesStatus;
    this._lastHesCodeChangeDate = lastHesCodeChangeDate;
  }
  
}